scripts/solve_network.py

```diff
@@ -555,24 +576,23 @@ def solve_network(n, config, opts="", **kwargs):
         )
     configure_logging(snakemake)
 
-    tmpdir = snakemake.params.solving.get("tmpdir")
-    if tmpdir is not None:
-        Path(tmpdir).mkdir(parents=True, exist_ok=True)
+    # tmpdir = snakemake.params.solving.get("tmpdir")
+    # if tmpdir is not None:
+    #     Path(tmpdir).mkdir(parents=True, exist_ok=True)
     opts = snakemake.wildcards.opts.split("-")
     solve_opts = snakemake.params.solving["options"]
 
     n = pypsa.Network(snakemake.input[0])
-    if snakemake.params.augmented_line_connection.get("add_to_snakefile"):
-        n.lines.loc[n.lines.index.str.contains("new"), "s_nom_min"] = (
-            snakemake.params.augmented_line_connection.get("min_expansion")
-        )
-    n = prepare_network(n, solve_opts)
-
+    # TODO Double-check handling the augmented case
+    # if snakemake.params.augmented_line_connection.get("add_to_snakefile"):
+    #    n.lines.loc[
+    #        n.lines.index.str.contains("new"), "s_nom_min"
+    #    ] = snakemake.params.augmented_line_connection.get("min_expansion")
+    n = prepare_network(n, solve_opts, config=solve_opts)
     n = solve_network(
         n,
         config=snakemake.config,
-        opts=opts,
-        solver_dir=tmpdir,
+        solving=snakemake.params.solving,
         solver_logfile=snakemake.log.solver,
```

Comment:
Please revise as discussed:
```suggestion
        log_fn=snakemake.log.solver,
```